holidays/countries/saint_martin.py

```
@@ -0,0 +1,48 @@
+    parent_entity = France
+    subdivisions = ()  # Override France subdivisions.
+    subdivisions_aliases = {}  # Override France subdivisions aliases.
+    # Cession from Guadeloupe on July 15th, 2007.
+    start_year = 2008
+
+    def _populate_public_holidays(self) -> None:
+        self.subdiv = "MF"
+        super()._populate_public_holidays()
+
+
+class SaintMartin(HolidaysMF):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)